interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -426,12 +435,19 @@ namespace {
         }
       }
 #else // _MSC_VER
+      const char* RootInclude = std::getenv("ROOT_INCLUDE");
```

Comment:
That would be great and makes much more sense indeed.
Is it something that is already implemented or does it need to be?